package com.example.demoStep3;

import org.springframework.boot.ApplicationRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;

import javax.sql.DataSource;
import java.sql.Connection;

@SpringBootApplication
public class DemoStep3Application {

	public static void main(String[] args) {
		SpringApplication.run(DemoStep3Application.class, args);
	}

    @Bean
    public ApplicationRunner runner(DataSource dataSource) {
        return args -> {
            Connection connection = dataSource.getConnection();
            System.out.println("DB 연결 성공: " + connection.getMetaData().getURL());
        };
    }
}
